config/config.go

```diff
@@ -122,3 +123,20 @@ func (c *Config) Validate() error {
 	}
 	return nil
 }
+
+// PubKeyEd25519 is type used to parse value provided by envconfig.
+type PubKeyEd25519 ed25519.PubKeyEd25519
+
+func (key *PubKeyEd25519) Decode(value string) error {
```

Comment:
Nice fix ;)

Reply:
Maybe we could do a PR on tendermint repo to add this function ;)

Reply:
They use viper for receiving flags/args/envs and so I don't think so it would pass :/